bedrock/firefox/templates/firefox/new.html

```diff
@@ -53,21 +53,18 @@
 
 <main role="main">
   <div class="version-message-container">
-    {% if l10n_has_tag('ios_announce') %}
-      <div id="version-message-ios" class="version-message">
-        {{_('Firefox is coming soon to iOS! <a href="{url}">Sign up to learn more</a>.').format(url=url('newsletter.ios'))|safe }}
-      </div>
-    {% else %}
-      <div id="version-message-ios" class="version-message">
-        {{_('Sorry, Firefox is currently unsupported by iOS devices.')}}
-      </div>
-    {% endif %}
     <div id="version-message-android-latest" class="version-message">
       {{_('Congrats! You’re using the latest version of Firefox.')}}
     </div>
     <div id="version-message-android-old" class="version-message">
       {{_('Looks like you’re using an older version of Firefox. Update on Google Play.')}}
     </div>
+    <div id="version-message-ios-latest" class="version-message">
+      {{_('Congrats! You’re using the latest version of Firefox.')}}
+    </div>
+    <div id="version-message-ios-old" class="version-message">
```

Comment:
I'm not too sure to be honest, best check with Cory in the bug on what the behaviour should be here? I think we are likely looking to avoid adding new strings if possible.